Source/WTF/wtf/URL.cpp

```diff
@@ -238,6 +238,27 @@ StringView URL::fragmentIdentifier() const
     return StringView(m_string).substring(m_queryEnd + 1);
 }
 
+// https://wicg.github.io/scroll-to-text-fragment/#the-fragment-directive
+String URL::consumefragmentDirective()
+{
+    ASCIILiteral fragmentDirectiveDelimiter = ":~:"_s;
+    auto fragment = fragmentIdentifier();
+    
+    if (fragment.isEmpty())
+        return nullString();
+    
+    auto fragmentDirectiveStart = fragment.find(StringView(fragmentDirectiveDelimiter));
```

Comment:
Calling the StringView constructor here isn't necessary.  StringView has a find that takes an ASCIILiteral.